common/script/content/spells.js

```diff
@@ -0,0 +1,499 @@
+import t from './translation';
```

Comment:
no reason. I was ready to do it directly in develop then I had to make a change for https://github.com/HabitRPG/habitrpg/pull/6794/files#diff-24cac23322db42dce4ea110a133c9ea3R221 so I did it into this branch. I'll make sure to move the changes to develop before merging